watertap/property_models/NaCl_prop_pack.py

```diff
@@ -749,26 +824,37 @@ def _enth_mass_phase(self):
 
         def rule_enth_mass_phase(
             b,
-        ):  # specific enthalpy, eq. 55 and 43 in Sharqawy  # TODO: remove enthalpy when all units can be isothermal
+        ):  # specific enthalpy, eq. 25 and 26 in Nayar et al. (2016)  # TODO: remove enthalpy when all units can be isothermal
             t = (
                 b.temperature - 273.15 * pyunits.K
             )  # temperature in degC, but pyunits in K
-            S = b.mass_frac_phase_comp["Liq", "NaCl"]
-            h_w = (
+            S = b.mass_frac_phase_comp["Liq", "TDS"]
```

Comment:
```suggestion
            S = b.mass_frac_phase_comp["Liq", "NaCl"]
```

Not sure why the index changed here...